crates/stackable-operator/src/commons/authentication/oidc.rs

```diff
@@ -111,10 +119,8 @@ impl AuthenticationProvider {
         }
     }
 
-    /// Returns the OIDC endpoint [`Url`]. To append the default OIDC well-known
-    /// configuration path, use `url.join()`. This module provides the default
-    /// path at [`DEFAULT_OIDC_WELLKNOWN_PATH`].
-    pub fn endpoint_url(&self) -> Result<Url> {
+    /// Base [`Url`] without any path set (so only protocol, host, port and such).
```

Comment:
suggestion: Slight rewording.

```suggestion
    /// Returns the base [`Url`] without any path segments.
    ///
    /// The base url only contains the scheme, the host, and an optional port.
```